### ISSUE TITLE
Error: error: function pg_catalog.pg_relation_is_updatable(oid, boolean) does not exist

### ISSUE THREAD

**Issue by cellis** [calebmer/postgraphql#298]
I'm trying to use this library with express but I keep getting this error:

```
error: function pg_catalog.pg_relation_is_updatable(oid, boolean) does not exist
at Connection.parseE (/Users/cameronellis/work/data-project/node_modules/pg/lib/connection.js:554:11)
    at Connection.parseMessage (/Users/cameronellis/work/data-project/node_modules/pg/lib/connection.js:381:17)
    at Socket.<anonymous> (/Users/cameronellis/work/data-project/node_modules/pg/lib/connection.js:117:22)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at TCP.onread (net.js:548:20)
```

I checked my database and, indeed, the `pg_catalog.pg_relation_is_updateable` routine doesn't appear in the list of routines. Is there any reason this is happening?

**Comment by calebmer** [comment]
What version of Postgres are you using?

**Comment by cellis** [comment]
I'm on 9.6.1

**Comment by benjie** [comment]
It looks like pg_relation_is_updatable was added [here](https://github.com/postgres/postgres/commit/dc3eb5638349e74a6628130a5101ce866455f4a3) in 2013; I'm not sure why it wouldn't be showing up - perhaps it's hidden behind a build option for postgresql?

**Comment by cellis** [comment]
Yeah, I tried looking up this function, there's nothing on google. I assume postgraphql is using it to introspect and create relational types.

**Comment by cellis** [comment]
I figured this out, I think it was related to having an older version of postgres 9.1 installed ( using brew AND postgres.app simultaneously may have had something to do with it? ). I reinstalled postgres.app and uninstalled the brew postgresql `brew uninstall postgresql`. The new postgres.app database `pg_catalog` has **2811** routines, including `pg_relation_is_updateable`, while the old one had around 2300. Thanks for taking the time to look into this.
